samcli/commands/init/command.py

```diff
@@ -240,6 +240,11 @@ def wrapped(*args, **kwargs):
     default=None,
     help="Enable AWS X-Ray tracing for your lambda functions",
 )
+@click.option(
+    "--application-insights/--no-application-insights",
+    default=None,
+    help="Enable CloudWatch Application Insights monitoring for your application",
```

Comment:
"Enable AWS CloudWatch Application Insights". has the help text been bar raised with UX? Its a crucial component on how customers understand and interpret.

Reply:
Yes, it was discussed and approved by UX